docs/_lab/04-running-scripts/index.md

```diff
@@ -25,7 +25,7 @@ Run the [bashcheck](https://github.com/hannob/bashcheck) script to check on Shel
 Download the `bashcheck.sh` script using `curl`, `wget`,  or similar:
 
 ```shell
-curl -O {{ site.url }}{{ site.baseurl }}/lessons/lesson1-10/src/bashcheck.sh
+curl -o src/bashcheck.sh https://raw.githubusercontent.com/hannob/bashcheck/master/bashcheck
```

Comment:
It seems like we thought we could serve both the bash and powershell script in https://github.com/puppetlabs/bolt/tree/master/docs/_includes/lesson1-10/src  at https://puppetlabs.github.io/bolt/lessons/lesson1-10/src/bashcheck.sh
 Is that not possible? We could have people just curl the github URL so 
```
curl -O https://raw.githubusercontent.com/puppetlabs/bolt/master/docs/_includes/lesson1-10/src/bashcheck.sh
```
Also note that the `-O` option saves the file with the remote name (which is what we want) the `-o` option expects a filename. It seems like this is the only place we really want/need to curl for some source code as the powershell script seems to be pretty short.